script.js

```diff
@@ -9,123 +11,71 @@ $(document).ready(function () {
             url: randomDrinkURL,
             method: "GET"
         }).then(function (response) {
-            // uses jQuery to grab randomdrinktitle section
-            let randomDrinkTitle = $('#randomdrinktitle');
-            // uses jQuery to grab randomText section
-            let randomText = $("#randomText");
-            // variable for if drink is alcoholic or not
-            let isAlcoholic = response.drinks[0].strAlcoholic
-            console.log(isAlcoholic);
-            if (isAlcoholic != "Alcoholic") {
-                randomDrinkTitle = "";
-                randomText = "";
-                randomDrink();
```

Comment:
This was a nice use of recursion! Unfortunately, there's a bug here.

When randomDrink() is called on line 22, it will go and set all the values for an alcoholic drink. However, once that finishes, execution will pick back up at line 23 and just overwrite those values with the non alcoholic drink.